smoderp2d/providers/base/__init__.py

```diff
@@ -112,6 +113,23 @@ def add_logging_handler(handler, formatter=None):
         handler.setFormatter(formatter)
         Logger.addHandler(handler)
 
+    def __load_hidden_config(self):
+        # load hidden configuration with advanced settings 
+        _path = os.path.join(os.path.dirname(__file__), '../../.config.txt')
```

Comment:
```suggestion
        _path = os.path.join(os.path.dirname(__file__), '..', '..', '.config.txt')
```

Reply:
I would suggest to use `.config.ini` as filename

Reply:
OK. changed it in commit 2884132d459968022ad277607eb47688595c56c3